### ISSUE TITLE
nix-indent-line messes up an attrset in a list in an attrset.

### ISSUE THREAD

**Issue by grahamc** [NixOS/nix-mode#60]
Given:

```nix
{
  x86 = [
    {
      name = "t1.small.x86";
    }
  ];
}
```

it becomes:

```nix
{
  x86 = [
  {
    name = "t1.small.x86";
  }
  ];
}
```

**Comment by grahamc** [comment]
I think this is a general confusion around how to handle indentation of these symbols:

```
let
  mozilla-overlay =
    import
  (
    builtins.fetchTarball
    https://github.com/mozilla/nixpkgs-mozilla/archive/master.tar.gz
  );
in mozilla-overlay
```

**Comment by grahamc** [comment]
Fixed by #61 and checked in #63
